src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -317,6 +317,8 @@ enum AppStrings {
 		static let onboardingInfo_togetherAgainstCoronaPage_title = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_title", comment: "")
 		static let onboardingInfo_togetherAgainstCoronaPage_boldText = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_boldText", comment: "")
 		static let onboardingInfo_togetherAgainstCoronaPage_normalText = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_normalText", comment: "")
+		static let onboardingInfo_togetherAgainstCoronaPage_Link = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_Link", comment: "URL")
```

Comment:
```suggestion
		static let onboardingInfo_togetherAgainstCoronaPage_link = NSLocalizedString("OnboardingInfo_togetherAgainstCoronaPage_Link", comment: "URL")
```

Reply:
I even suggest the camel format for our project.

Reply:
included this suggestion.